_posts/2017-02-13-how-to-run-multiple-dependency-builds-on-your-ci-server.markdown

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title:  "How to run multiple dependent builds on Circle CI"
+date: 2017-02-20 15:21:00
+categories: ["continuous-integration, circle-ci"]
+author: "luciano"
+---
+
+We all know the importance of having in your projects a [CI](https://en.wikipedia.org/wiki/Continuous_integration) tool integrated in your project. It allows you to run your entire test suite every time you want to merge a set of changes.
+If you have a core project and many projects that depend on it, you want to run the tests for the core project and the dependent projects at the same time. This article explains how you can do it with [Circle CI](https://circleci.com).
+
+<!--more-->
+
+I will explain how to do that with two Rails projects (an application and an [engine](http://guides.rubyonrails.org/engines.html))
```

Comment:
@lubc Please add a link to Rails in "two Rails projects" -> http://rubyonrails.org